content/blog/2017/2017-04-10-Jenkins-has-upgraded-to-Java-8.adoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Jenkins Now Requires Java 8 To Run Starting With 2.54"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+:imagesdir: /images/post-images/2017-04-java8
+
+image:lambda.png[role="right"] As link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced some weeks ago], the necessary changes have now been merged into Jenkins codebase.
+That means that starting with link:https://jenkins.io/changelog/#v2.54[Jenkins **2.54**], the Java runtime required for Jenkins is updated to Java 8.
+
+== Timeline
+
+* Earlier this year: announce this upgrade.
+* April, 2017: Drop support for Java 7 in Jenkins weekly: **2.54**.
+* June 2017: First LTS version requiring Java 8 is published.
+
+== FAQ
+
+I have only a JDK7 available, can I still run Jenkins 2.54+ with it?::
+No, you need to install a JDK8.
+
+Can I still run my build agents with Java 7?::
+No. A build agent must run at least the Java version which the Jenkins Master was compiler for.
+This means you must also upgrade your build agents JVM if not already done.
+
+== References
+
+* link:/blog/2016/11/22/what-jvm-versions-are-running-jenkins-the-return/[JVM statistics post back in November 2016]
+* link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[Official announce on the Jenkins Blog]
+* Original link:https://issues.jenkins-ci.org/browse/JENKINS-27624[JIRA ticket for this issue]
+* The link:https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jenkinsci-dev/fo5nKLhZK5U/Zb9jHpBJBQAJ[6 months, 82 messages, thread on the Jenkins developers mailing list]
+* The link:https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jenkinsci-users/R6SGnW7MY6A/25MBJ9a2CQAJ[announcement on the Jenkins users mailing list]
+* The link:https://github.com/jenkinsci/jenkins/pull/2802[Pull request on _Jenkins core_]
```

Comment:
Which line do you target with this comment? All above, or only the link to the PR? 

If the latter, I tend to disagree. But well, this is easily findable with the JIRA link, so can remove. If the former, please clarify.